packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -4,186 +4,237 @@ excerpt: Configure your Okta org and your server-side application to secure your
+Add a layer of authorization to your web services with [Okta API Access Management](/docs/concepts/api-access-management/).
 
 ---
 
 **Learning outcomes**
 
-* Add Okta [authorization](https://www.okta.com/identity-101/authentication-vs-authorization/) to your API endpoints.
-* Add dependencies and configure your API.
-* Create an anonymous endpoint and restricted endpoint.
-* Require authorization on protected routes.
-* Make an HTTP request with and without a valid access token.
+* Configure a web API to use Okta
+* Define which endpoints require authorization and which don't
+* Enable Cross-Origin Resource Sharing (CORS) for the API
+* Test the API is secure
 
 **What you need**
 
-A [custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server) to create and apply authorization policies to protect your APIs
-
-<ApiAmProdWarning />
+* An [Okta Developer Edition org](https://developer.okta.com/signup/)
+* [Postman](https://www.getpostman.com/apps) to test the API
+* <StackSnippet snippet="whatyouneed" />
 
 **Sample code**
 
 <StackSnippet snippet="samplecode" />
 
+> **Note**: Several standalone tools can send requests to APIs and allow you to inspect the responses. Our documentation uses **Postman** and offers [Postman Collections](/docs/reference/postman-collections/) to test its APIs more efficiently with a GUI. It also includes HTTP requests as text for those who prefer to use a terminal utility such as [cURL](https://curl.se/download.html).
+
 ---
 
-## Set up Okta
+## Overview
 
-Set up your developer account and [Okta org](/docs/concepts/okta-organizations/). The Okta command-line interface (CLI) is the quickest way to do this. Alternatively, you can [manually sign up for a developer account](https://developer.okta.com/signup/).
+Background services and third-party APIs accessing your own APIs require the same level of [authentication and authorization](https://www.okta.com/identity-101/authentication-vs-authorization/) as users accessing your web applications. However, a machine-to-machine sign-in flow should be silent and require no human user interaction. Use Okta to grant the correct level of access to your APIs on your behalf.
 
-1. Install [Okta CLI](https://cli.okta.com/).
-1. If you don't already have a free Okta developer account:
-   1. Open your terminal.
-   {style="list-style-type:lower-alpha"}
-   1. Run `okta register`, and enter your first name, family name, email address, and country.
-   1. Click or tap **Activate** in the account activation email that is sent to the email address that you gave.
+In this quickstart, you:
 
-      > **Tip**: If you don't receive the confirmation email sent as part of the creation process, check your spam filters for an email from `noreply@okta.com`
+1. [Check that API Access Management is enabled](#check-that-api-access-management-is-enabled)
```

Comment:
```suggestion
1. [Check that API Access Management is enabled](#check-that-api-access-management-is-enabled).
```

Reply:
I see that these are links to headings in the same doc, so maybe periods aren't ideal here.  However, I'm adding them because it's an ordered list with items that complete the stem. If you want to leave off the periods, recommend changing the stem to introduce this as a workflow sequence (so it's clear that these are exact headings and not just hyperlinked phrases).

Maybe something like 

This quickstart contains the following tasks: 

(that would also solve the translation recommendation in the previous comment)
@susanharper-okta for style guide visibility

Reply:
I've gone with the workflow sequence option you suggested here and left the periods off the links.